Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
+gem "cocoapods", "1.6.0.beta.1"
```

Comment:
I really don't like using beta software for library releases. Do not force specific cocoapods version on the end user who might just be trying the library.

Reply:
Let's use CocoaPods 1.5.3.

Reply:
@dzindra What about CocoaPods 1.6.0.rc.1?